distribution/lib/Standard/Base/0.0.0-dev/src/Data/Decimal.enso

```diff
@@ -525,6 +525,29 @@ type Decimal
     negate : Decimal
     negate self = Decimal.Value self.big_decimal.negate
 
+    ## GROUP CONVERSIONS
+       Convert this to an `Integer`.
+
+       > Example
+         Convert `Decimal` 2345 to an `Integer`.
+             d = Decimal.new "2345"
+             d.to_integer
+             # => 2345
```

Comment:
I'd like us to have an example that shows some more interesting case, e.g. `2345.6` to see what happens with the decimal part? Is it truncated, rounded? Is it only possible to convert if decimal part is `.0` and it fails otherwise?

Reply:
Done. (It succeeds either way.)